README.md

```diff
@@ -0,0 +1,159 @@
+<!--
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+[![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+# vLLM Backend
+
+The Triton backend for [vLLM](https://github.com/vllm-project/vllm).
+You can learn more about Triton backends in the [backend
+repo](https://github.com/triton-inference-server/backend). Ask
+questions or report problems on the [issues
+page](https://github.com/triton-inference-server/server/issues).
+This backend is designed to run
+[supported models](https://vllm.readthedocs.io/en/latest/models/supported_models.html)
+on a
+[vLLM engine](https://github.com/vllm-project/vllm/blob/main/vllm/engine/async_llm_engine.py).
+
+This is a Python-based backend. When using this backend, all requests are placed on the
```

Comment:
Would be good to hyperlink "python-based backend" to the docs on it when https://github.com/triton-inference-server/backend/pull/88 is merged.

Reply:
We should normalize our terms:

python-based or python based

Reply:
Also, Python should be capitalized in my opinion. We capitalize Python in the [Python backend README](https://github.com/triton-inference-server/python_backend). Capitalizing the "p" in Python also aligns with the capitalization guidelines in [Python's style guide](https://devguide.python.org/documentation/style-guide/#capitalization).

Reply:
Noted

Reply:
are we consistent with 'P' capitalization in other places? Just double checking. As consistency within documentation would be more important

Reply:
Doing a search through the Triton Inference Server GitHub organization for markdown files with the word "Python" in them, we are pretty consistent with using capitalization. There are a few documents in the tutorials where we use both versions that we could update for consistency, e.g. [part 6 in tutorials](https://github.com/triton-inference-server/tutorials/blob/af67595b38c30738a4bdf356bc26e96203284139/Conceptual_Guide/Part_6-building_complex_pipelines/README.md?plain=1#L73) and the new [request cancellation document](https://github.com/triton-inference-server/server/blob/328d51940c3402f3594bcab9c2c6155740cf043f/docs/user_guide/request_cancellation.md?plain=1#L100).

Reply:
`Python-based backend` it is